packages/common/rollup-configs.js

```diff
@@ -20,46 +21,79 @@ const getExternalModuleTester = (pkg) => {
   return (moduleID) => externalModuleTests.some((regexp) => regexp.test(moduleID));
 };
 
-const getBanner = (pkg) => `/*
+const getBanner = (pkg, customText) => {
+  const text = `
   OpenShift Dynamic Plugin SDK
   ${pkg.repository.url.replace(/\.git$/, '')}
 
   ${pkg.name} version ${pkg.version}
-  ${new Date().toLocaleString('en-US', { dateStyle: 'long', timeStyle: 'long' })}
-  commit ${execSync('git rev-parse HEAD').toString().trim()}
-*/\n`;
+  build date ${new Date().toLocaleString('en-US', { dateStyle: 'long', timeStyle: 'long' })}
+  git commit ${execSync('git rev-parse HEAD').toString().trim()}
+
+  ${customText ?? ''}
+  `.trim();
+
+  return `/*\n  ${text}\n */\n`;
+};
 
 /**
+ * @returns {import('rollup').Plugin}
+ */
+const replaceLodashEsRequire = () => ({
+  name: 'replace-lodash-es-require',
+
+  renderChunk(code, chunk) {
+    return {
+      code: code.replace("require('lodash-es')", "require('lodash')"),
```

Comment:
This will not work for direct imports, better to change this to less strict replace
```
code.replace("require('lodash-es", "require('lodash")
```

Reply:
This is also the reason why there's no `lodash-es` in `lib-webpack`'s `dependencies`.

@karelhala @florkbr In each "lib" package, `peerDependencies` and `dependencies` reflect the consumer side of the package.

Reply:
@vojtechszocs I was more concerned about direct and named imports. If we were to use
```JS
import defaultsDeep from 'lodash-es/defaultsDeep'
```

That would be transplied as `var defaultsDeep = require('lodash-es');` but we can discuss the usage of this later down the line. No need to stall this PR on this anymore.